gax/src/main/java/com/google/api/gax/rpc/ErrorDetails.java

```diff
@@ -0,0 +1,38 @@
+package com.google.api.gax.rpc;
+
+import com.google.auto.value.AutoValue;
+import com.google.auto.value.extension.toprettystring.ToPrettyString;
+import java.util.List;
+import java.util.Map;
+
+@AutoValue
+public abstract class ErrorDetails {
```

Comment:
Please add Javadoc for the class and its methods (it is a public class and it is very likely that any user of this class will want to know what each method means when they debug error details received from the server).

Reply:
Added